modules/custom/az_migration/src/Plugin/migrate/process/EntityEmbedProcess.php

```diff
@@ -208,8 +209,8 @@ public function updateEmbedTag($id, $type, $tag, $view, \DOMDocument $dom, \DOME
    */
   public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
 
-    $dom = new \DOMDocument();
-    $dom->loadHTML($value, LIBXML_HTML_NOIMPLIED | LIBXML_HTML_NODEFDTD);
+    $html5 = new HTML5(['disable_html_ns' => TRUE]);
+    $dom = $html5->loadHTML($value);
```

Comment:
```suggestion
    $dom = $html5-> loadHTMLFragment($value);
```

I think this would prevent the root level `<html>` element from getting added.

Reply:
I did try that, and it caused other errors, so I'm not sure if that will work.

Reply:
FYI, here is the document I am copying from https://git.drupalcode.org/project/media_migration/-/blob/8.x-1.x/src/Plugin/migrate/process/ImgTagToEmbedFilter.php